_config.yml

```diff
@@ -33,30 +33,32 @@ permalink: pretty
 logo: "/assets/images/FeatureBase-Logo-Gradient-Wide.png"
 
 color_scheme: featurebase # file URL: /_sass/color_schemes/featurebase.css
-
 # Enable or disable the site search
 # Supports true (default) or false
 search_enabled: true
-
-# Split pages into sections that can be searched individually
-# Supports 1 - 6, default: 2
-search.heading_level: 2
-
-# Maximum amount of previews per search result
-# Default: 3
-search.previews: 10
-
-# Maximum amount of words to display before a matched word in the preview
-# Default: 5
-search.preview_words_before: 5
-
-# Maximum amount of words to display after a matched word in the preview
-# Default: 10
-search.preview_words_after: 10
-
-# Enable or disable the search button that appears in the bottom right corner of every page
-# Supports true or false (default)
-search.button: true
+search:
+  # Split pages into sections that can be searched individually
+  # Supports 1 - 6, default: 2
+  heading_level: 2
+  # Maximum amount of previews per search result
+  # Default: 3
+  previews: 2
```

Comment:
Let's turn this back to 10. I see why more search results in the site right now than I do this build. We don't want search restricts

Reply:
ARGH! Sorry! Fixing